heroku/cli
# migrate(W-14179873): pg-v5: Upgrade pg:settings:auto-explain:log-verbose

This PR moves the `pg:settings:auto-explain:log-verbose` command to the CLI and migrates it to TypeScript

### Test Instructions
1. Checkout this branch and run `yarn && yarn build && cd packages/cli`
2. Run `./bin/run pg:settings:auto-explain:log-verbose --app YOUR_APP DATABASE`
3. Verify the output is the same as the original.
